scripts/create_lightning_rc.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+OLDVER=0.37.0
+LVER=0.38.0
+rreplace(){
+   grep -rl "$1" . | xargs sed -i "s|$1|$2|g"
+}
+
+# clean up and checkout master
+git reset . && git checkout .
+git fetch
+git checkout master
+git pull
```

Comment:
do we need to pull from GH origin?
```suggestion
git pull origin master
```